docs/source/contribution/technical_steering_committee.md

```diff
@@ -8,12 +8,12 @@ The TSC is responsible for the project's future development; you can read about
 
 On this page we describe:
 
-- [Responsibilities of a maintainer](#responsibilities-of-a-maintainer)
-- [Requirements to become a maintainer](#requirements-to-become-a-maintainer)
-- [Current maintainers](#current-maintainers)
-- [Past maintainers](#past-maintainers)
-- [Application process](#application-process)
-- [Voting process](#voting-process)
+ - [Responsibilities of a maintainer](#responsibilities-of-a-maintainer)
+ - [Requirements to become a maintainer](#requirements-to-become-a-maintainer)
+ - [Current maintainers](#current-maintainers)
+ - [Past maintainers](#past-maintainers)
+ - [Application process](#application-process)
+ - [Voting process](#voting-process)
```

Comment:
Are these changes intentional?

Reply:
I usually just go with "What would Prettier do?" (even though it doesn't seem to be enabled here), and this is not consistent with the rest of the file. Pretty minor, but also kinda annoying when want to blame something and need to go through an extra unnecessary change. :P If you don't mind. :)

Reply:
I thought I reversed this 🤦

Reply:
Went ahead and did it, and reenabled auto-merge.